src/Morphir/Type/MetaTypeMapping.elm

```diff
@@ -35,12 +35,15 @@ lookupConstructor baseVar refs ((FQName packageName moduleName localName) as fQN
                         (\( typeName, typeSpec ) ->
                             case typeSpec.value of
                                 Type.CustomTypeSpecification paramNames ctors ->
-                                    case ctors |> Dict.get localName of
-                                        Just ctorArgs ->
-                                            [ ctorToMetaType baseVar refs (MetaRef (FQName packageName moduleName typeName)) paramNames (ctorArgs |> List.map Tuple.second) ]
+                                    ctors
+                                        |> List.filterMap
+                                            (\(Type.Constructor ctorName ctorArgs) ->
+                                                if ctorName == localName then
+                                                    Just (ctorToMetaType baseVar refs (MetaRef ( packageName, moduleName, typeName )) paramNames (ctorArgs |> List.map Tuple.second))
 
-                                        Nothing ->
-                                            []
+                                                else
+                                                    Nothing
+                                            )
 
```

Comment:
There seems to be a merge issue here. You are trying to overwrite my changes which is why the build fails.